packages/paste-design-tokens/tokens/global/font-family.yml

```diff
@@ -8,3 +8,11 @@ props:
     value: "{!font-family-10}"
   font-family-code:
     value: "{!font-family-20}"
+  font-family-text-japanese:
+    value: "{!font-family-30}"
+  font-family-text-korean:
+    value: "{!font-family-40}"
+  font-family-text-chinese-traditional:
+    value: "{!font-family-50}"
+  font-family-text-chinese-simplified:
+    value: "{!font-family-60}"
```

Comment:
praise: I really like this approach and think it's going to be useful if we need flexibility in the future. So good!